.travis.yml

```diff
@@ -8,11 +8,10 @@ php:
   - nightly
 
 install:
-  - wget https://phar.phpunit.de/phpunit-old.phar
   - composer install
 
 script:
-  - php phpunit-old.phar
```

Comment:
Phpunit its actually a dependency even if you try to hide it not including it directly in the composer.json. Its better use the version that composer install in `vendor\bin` to make composer the only required required to run the project which its the facto standard.